app/views/organizations/index.html.erb

```diff
@@ -1,14 +1,11 @@
 <div class="container">
   <div class="row">
-
       <div class="col-sm-5 col-lg-7">
         <% @organizations.each do |organization| %>
             <section id="facility-groups">
               <div class="card" title="<%=organization.name%>">
                 <h2><%= organization.name %></h2>
-
-
-                    <% policy_scope([:cohort_report, organization.facility_groups])
+                    <% current_admin.accessible_facility_groups(:view_reports).where(organization: organization)
```

Comment:
This is potentially a bad first example to migrate because it does some facility.district weirdness.
I've made the most literal translation to the new permissions so its comparable. Might end up refactoring this quite a bit.